wordpress-mobile/WordPress-iOS
# Adds a debug menu and overriding feature flags

This PR adds a Debug menu option to the bottom of App Settings (only displayed in debug builds), and allows you to toggle feature flags on and off at runtime.

![ff1](https://user-images.githubusercontent.com/4780/69752831-035e8680-114a-11ea-9cd7-5cca0d36aa4f.gif)

Of course, depending on the implementation of feature flagged code, it's not guaranteed that toggling a flag while the app is already running will work correctly. But worst case, you should be able to restart the app after toggling a flag.

**To test**

* Build and run
* Domain credits is probably the easiest to test (or add your own simple feature flag!). Find a site that has a domain credit (or purchase a plan for a test site), and on the blog details screen observe that the domain credit banner is present.
* Navigate to Me > App Settings, and scroll to the bottom to Debug
* Toggle off the domain credits feature flag
* Go back to blog details and check the domain credit banner has gone
* Go back to the Debug screen and turn the feature flag back on
* Go back to blog details and check the banner is back

PR submission checklist:
